packages/components/psammead-caption/src/index.stories.jsx

```diff
@@ -41,8 +41,9 @@ storiesOf('Components|Caption', module)
       ],
       ([linkText, captionText], script) => (
         <Caption script={script}>
-          {captionText}
-          <InlineLink href="https://www.bbc.com"> {linkText}</InlineLink>.
+          {`${captionText} `}
+          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>.
```

Comment:
```suggestion
          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>
```

remove hardcoded fullstop